aiidalab_launch/__main__.py

```diff
@@ -113,6 +114,15 @@ def cli(verbose):
 
     LOGGER.info(f"Configuration file path: {_application_config_path()}")
 
+    latest_version = get_latest_version(timeout=0.1)
+    if latest_version and latest_version > parse(__version__):
+        click.secho(
+            f"A new version of aiidalab-launch is available: {latest_version} (installed: {__version__})",
+            fg="yellow",
+        )
+        if "pipx" in __file__:
```

Comment:
Just curious when this is `True`? I tried installing `pipx` both globally and in my environment and this message doesn't seem to pop up.

Reply:
This should be True when aiidalab-launch is installed via pipx. I've tested this by manually force installing it from the repository with a manipulated `__version__`: `pip install /path/to/aiidalab-launch -f`.

Reply:
Yup, seems to work! 👍 All good then.